client/src/app/components/stardust/history/TransactionHistory.tsx

```diff
@@ -37,94 +44,50 @@ const TransactionHistory: React.FC<TransactionHistoryProps> = (
-
-                            if (!transactionId) {
-                                return null;
-                            }
-
-                            // rotate row background colour for different transaction ids
-                            if (idx > 0) {
-                                const previousItemDetails = outputDetailsMap[historyView[idx - 1].outputId];
-                                const previousSpent = historyView[idx - 1].isSpent;
-                                if (previousItemDetails) {
-                                    const previousTransactionId = previousSpent ?
-                                        previousItemDetails.metadata.transactionIdSpent :
-                                        previousItemDetails.metadata.transactionId;
-
-                                    if (transactionId !== previousTransactionId) {
-                                        isDarkBackgroundRow = !isDarkBackgroundRow;
-                                    }
-                                }
-                            }
-
-                            return (
-                                <React.Fragment key={idx}>
-                                    <TransactionRow
-                                        outputId={historyItem.outputId}
-                                        transactionId={transactionId}
-                                        date={historyItem.milestoneTimestamp}
-                                        milestoneIndex={historyItem.milestoneIndex}
-                                        value={Number(outputDetails.output.amount)}
-                                        isSpent={historyItem.isSpent}
-                                        isFormattedAmounts={isFormattedAmounts}
-                                        setIsFormattedAmounts={setIsFormattedAmounts}
-                                        darkBackgroundRow={isDarkBackgroundRow}
-                                    />
-                                </React.Fragment>
-                            );
-                        })
-                    )}
+                    {transactions?.map((c, idx) => (
+                        <React.Fragment key={idx}>
+                            <TransactionRow
+                                isGenesisByDate={c.isGenesisByDate}
+                                isTransactionFromStardustGenesis={c.isTransactionFromStardustGenesis}
+                                transactionLink={c.transactionLink}
+                                dateFormatted={c.dateFormatted}
+                                balanceChangeFormatted={c.balanceChangeFormatted}
+                                transactionId={c.transactionId}
+                                isSpent={c.isSpent}
+                                isFormattedAmounts={isFormattedAmounts}
+                                setIsFormattedAmounts={setIsFormattedAmounts}
+                                darkBackgroundRow={idx % 2 === 0}
```

Comment:
I would remove this logic, we dont need it anymore and the other tables in the explorer, if they dont need distinction, are all the same color

Reply:
This is how its now
![image](https://github.com/iotaledger/explorer/assets/3624944/124419f9-537b-4832-8bbf-a3666c82071f)

And this is what I suggest
![image](https://github.com/iotaledger/explorer/assets/3624944/c54c766c-a914-49d6-bea9-594167b8b8f0)